Content.Shared/Throwing/ThrowAttemptEvent.cs

```diff
@@ -13,6 +13,9 @@ public ThrowAttemptEvent(EntityUid uid, EntityUid itemUid)
         public EntityUid ItemUid { get; }
     }
 
+    [ByRefEvent]
+    public record struct ThrowItemAttemptEvent(EntityUid User, bool Cancelled = false);
```

Comment:
I like to add what entity it's being raised on here so I don't need to go to the eventbus method (i.e. raised directed on the item being thrown).

Reply:
FIxed in [`c9cdd0d` (#30193)](https://github.com/space-wizards/space-station-14/pull/30193/commits/c9cdd0dbccfce89473b77f4de5eff9c544f8d5c9)